app/admin/Settings.scala

```diff
@@ -40,17 +42,15 @@ object Settings {
   def fromLocalFile(source: SettingsSource.LocalFile): Either[Throwable, Settings] =
```

Comment:
what's the difference between Settings and SettingsSource, as both seem to be able to get from a file?

Reply:
From looking at it `SettingsSource` provides the location and `Settings` does the actual read. I didn't write the original code so not 100% sure on why it was divided up this way